File: app/src/main/java/com/coppermobile/mysampleusingmvvmlivedata/repository/DishAPIInterface.java
package com.coppermobile.mysampleusingmvvmlivedata.repository;

import com.coppermobile.mysampleusingmvvmlivedata.data.source.remote.DishResponse;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface DishAPIInterface {

    @GET("json/menu.json")
    Call<List<DishResponse>> getDishes();
}

File: app/src/main/java/com/coppermobile/kotlinsamplemvvmlivedataroom/repository/DishAPIInterface.kt
package com.coppermobile.kotlinsamplemvvmlivedataroom.repository

import com.coppermobile.kotlinsamplemvvmlivedataroom.data.source.remote.DishResponse
import retrofit2.Call
import retrofit2.http.GET

interface DishAPIInterface {

    @GET("json/menu.json")
    fun getDishes(): Call<List<DishResponse>>?
}